src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -31,7 +31,7 @@ public class ProductController {
     private static final String PAGE = "0";
     private static final String SORT_FIELD = "title";
     private final ProductService productService;
-    private final DtoMapper<Product, ProductRequestDto, ProductResponseDto> productMapper;
+    private final ProductMapper productMapper;
```

Comment:
keep using interface to refer to mapper. realization may be in one or two class - controller doesn't care